_src/assets/js/get-avatar.js

```diff
@@ -0,0 +1,51 @@
+
+const fr = new FileReader();
+const uploadBtn = document.querySelector('.js__profile-trigger');
+const fileField = document.querySelector('.js__profile-upload-btn');
+const profileImage = document.querySelector('.js__profile-image');
+const profilePreview = document.querySelector('.js__profile-preview');
+
+
+/**
+ * Recoge el archivo añadido al campo de tipo "file"
+ * y lo carga en nuestro objeto FileReader para que 
+ * lo convierta a algo con lo que podamos trabajar.
+ * Añade un listener al FR para que ejecute una función
+ * al tener los datos listos
+ * @param {evento} e 
+ */
+function getImage(e){
+  var myFile = e.currentTarget.files[0];
```

Comment:
I think you could change the word `var` to `const`

Reply:
Should be fixed, working as intended!